lib/plugins/wallet/cardano/wallet.js

```diff
@@ -0,0 +1,67 @@
+const fs = require('fs')
+const { promisify } = require('util')
+
+const cardanoApi = require('./client')
+const crypto = require('crypto')
+
+const coinUtils = require('../../../coin-utils')
+const options = require('../../../options')
+
+const cryptoRec = coinUtils.getCryptoCurrency('ADA')
+const configPath = coinUtils.configPath(cryptoRec)
+
+function sha256(input) {
+  return crypto.createHash('sha256').update(input).digest('base64')
+}
+
+async function getCardanoMnemonic() {
+  return promisify(fs.readFile)(options.cardanoMnemonicPath, 'utf8')
+    .then(text => text.trim())
+    .then(mnemonicText => ({
+      mnemonic: mnemonicText.split(/[ \r\n]+/),
+      hash: sha256(mnemonicText)
+    }))
+}
+
+async function getWalletConfiguration() {
+  return promisify(fs.readFile)(configPath).then(text => JSON.parse(text))
+}
+
+async function saveWalletConfiguration(configuration) {
+  return fs.writeFile(configPath, JSON.stringify(configuration))
```

Comment:
I have seen usage of `common.writeFile` in some other modules can not (should not) it be used everywhere?